### PR TITLE
rewrite for good v7.x.x

### LINKED ISSUE #14: Update dependencies
Update dependencies including _good_ 7.x.x

### PR DESCRIPTION
**Pull request by cjihrig**
There is still one line of coverage needed. And I'm not sure what the `good` policy is now for filtering out the ops events that this module rejected in previous versions. But, I wanted to get eyes on it.

Closes #14

### CODE REVIEW

**Comment by arb** [comment]
This looks fine to me. Do we know the reason "ops" events were filtered out? Did loggly not like them for some reason or did this person just not want them?

**Comment by cjihrig** [comment]
I'm not sure that it's actually a big deal - https://github.com/continuationlabs/good-loggly/issues/11.

**Comment by arb** [comment]
Then you can probably remove that stuff entirely. If you don't ops going into loggly, then you'd filter it out somewhere else.

**Comment by cjihrig** [comment]
OK. Then I'm going to get coverage on the one missing line, and this should be good to go.

File: lib/index.js [hunk 1]
```diff
@@ -1,46 +1,51 @@
...
+      subdomain: config.subdomain,
+      json: true,
+      // Tags in JSON logs don't seem to appear properly in Loggly when
+      // sent with the X-LOGGLY-TAG header
+      useTagHeader: false,
+      tags: Array.isArray(config.tags) ? config.tags : []
+    });
   }
+  _write (event, encoding, callback) {
+    const data = Merge({}, event);
```
Comment (arb) [inline lib/index.js]: I don't THINK you need to do this anymore. Each reporter pipeline get's it's own copy so you are free to edit it as you see fit.

File: lib/index.js [hunk 2]
```diff
@@ -1,46 +1,51 @@
...
 
-    // Tags in JSON logs don't seem to appear properly in Loggly when sent with the X-LOGGLY-TAG header
-    useTagHeader: false,
-    tags: Array.isArray(config.tags) ? config.tags : []
-  });
-  this._config = Merge({}, config);
-  this._streams = {
-    squeeze: GoodSqueeze.Squeeze(events)
-  };
+    data.timestamp = GoodLoggly.timeString(data.timestamp);
```
Comment (arb) [inline lib/index.js]: Seems like a pointless extra call on the stack.

Comment (cjihrig) [inline lib/index.js]: Yea. I thought so too. It was clearly done for improved testability. I think we should leave it. Same with `GoodLoggly.getMessage()`.

File: lib/index.js (hunk 1 above)
Comment (cjihrig) [inline lib/index.js]: Removed

**Pull request merged**